components/picker/index.css

```diff
@@ -32,7 +32,6 @@ governing permissions and limitations under the License.
 	--spectrum-picker-spacing-bottom-to-text: var(--spectrum-component-bottom-to-text-100);
 	--spectrum-picker-spacing-edge-to-text: var(--spectrum-component-edge-to-text-100);
 	--spectrum-picker-spacing-edge-to-text-quiet: var(--spectrum-field-edge-to-text-quiet);
-	--spectrum-picker-spacing-top-to-text-side-label-quiet: var(--spectrum-component-top-to-text-100);
```

Comment:
Can you outline why these variables are okay to be removed in the PR description when you get a chance? I see that it's fine in the baselines but just want to have it documented for the future.